lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -385,6 +385,7 @@ sub overview {
                 jobid    => $job->id,
                 state    => "done",
                 failures => $job->failed_modules_with_needles(),
+                comments => scalar $job->comments,
             };
```

Comment:
this very innoncent line will make sure the overview page will downgrade from 4s to 30s loadtime.

if you check your log files, it will do tons of  "SELECT * from job_comments where job_id=..." - and causing a round trip for each and every job.

Prefetch this - and make sure you tested this code with a production database dump. Correctness isn't the only thing users care for ;)